assets/themes/default/_variables.scss

```diff
@@ -24,14 +24,20 @@ $margin: 0 $horizontal-margin;
 
 /* Fonts */
 
-$route159: "Route159", serif;
-$playfair: "Playfair Display", serif;
-$lora: "Lora", serif;
+$route159: "Vazir",
+"Route159",
+serif;
+$playfair: "Vazir",
+"Playfair Display",
+serif;
+$lora: "Vazir",
+"Lora",
+serif;
```

Comment:
VSCode Prettier extension did this.

Reply:
since i'm not sure it's that much "prettier", or more readable for that matter, could we put this back into one line each?

after that it should be good to merge

Reply:
fixed.